src/Messages/Attachments/TSAttachmentStream.h

```diff
@@ -16,8 +17,6 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithContentType:(NSString *)contentType NS_DESIGNATED_INITIALIZER;
 - (instancetype)initWithPointer:(TSAttachmentPointer *)pointer NS_DESIGNATED_INITIALIZER;
 
-@property (atomic, readwrite) BOOL isDownloaded;
```

Comment:
Once we merge https://github.com/WhisperSystems/Signal-iOS/pull/1849 we should be able to do away with this property altogether. Sound like a plan?